.teamcity/settings/extensions.kt

```diff
@@ -21,6 +21,17 @@ fun ParametrizedWithType.java8Home(os: Os) {
     param("env.JAVA_HOME", "%${os.name}.java8.oracle.64bit%")
 }
```

Comment:
Do we still need `java8Home()`? If so, I think it call through to `javaHome()`.

Reply:
I removed `java8Home`.